sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/policy/ScrubEtagPolicy.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common.policy;
+
+import com.azure.core.http.HttpHeader;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class ScrubEtagPolicy implements HttpPipelinePolicy {
+    private static final String ETAG = "eTag";
```

Comment:
I believe there is a constant in common for `eTag`

Reply:
ETAG in common is "ETag", what the response header has in name is "eTag".

Reply:
The `get` method on `HttpHeaders` uses a case insensitive implementation, so this should be fine. Additionally, `ETag` is the more correct implementation as that is what the documentation states is the header return name.

Reply:
Cannot find the ETAG anymore. Leave it as it is.